DigitalLearningSolutions.Web/Controllers/LoginController.cs

```diff
@@ -175,6 +175,12 @@ UserAccountSet accounts
                 IsPersistent = rememberMe,
                 IssuedUtc = DateTime.UtcNow
             };
+
+            if (!rememberMe)
+            {
+                authProperties.ExpiresUtc = DateTime.UtcNow.AddHours(8);
+            }

```

Comment:
Nice, could we just add a comment to explain why we're doing it? Most important to point to the bug ticket, but also just say that tutorial progress doesn't get updated if the auth token has expired by the end of the tutorial.